pkgs/by-name/cl/clash-verge-rev/unwrapped.nix

```diff
@@ -32,7 +34,11 @@ rustPlatform.buildRustPackage {
         --replace-fail "libayatana-appindicator3.so.1" "${libayatana-appindicator}/lib/libayatana-appindicator3.so.1"
 
       substituteInPlace $cargoDepsCopy/sysproxy-*/src/linux.rs \
-        --replace-fail '"gsettings"' '"${glib.bin}/bin/gsettings"'
+        --replace-fail '"gsettings"' '"${glib.bin}/bin/gsettings"' \
+        --replace-fail '"kreadconfig5"' '"${libsForQt5.kconfig}/bin/kreadconfig5"' \
+        --replace-fail '"kreadconfig6"' '"${kdePackages.kconfig}/bin/kreadconfig6"' \
+        --replace-fail '"kwriteconfig5"' '"${libsForQt5.kconfig}/bin/kwriteconfig5"' \
+        --replace-fail '"kwriteconfig6"' '"${kdePackages.kconfig}/bin/kwriteconfig6"'
```

Comment:
Do we really need qt5 and qt6?

Reply:
In case the user uses plasma 5. The sysproxy-rs lib determines the version by `KDE_SESSION_VERSION`, which will not try both version. The source code is [here](https://github.com/zzzgydi/sysproxy-rs/blob/45fffb0cdddb7c2d0ce30300dd72a934bb458c09/src/linux.rs#L239-L243).

There might be other ways, but I don't know for now.